libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -223,8 +232,6 @@ def _inject_state(
             BaseModel,
         ],
     ) -> ToolCall:
-        if tool_call["name"] not in self.tools_by_name:
-            return tool_call
         state_args = _get_state_args(self.tools_by_name[tool_call["name"]])
```

Comment:
We should be doing this once in the constructor and caching so we don't have to do all this meta-programming every time the node is called

Reply:
nice, good idea